cpp/src/stream_compaction/copy_if.cuh

```diff
@@ -352,6 +353,14 @@ gdf_column copy_if(gdf_column const &input, Filter filter,
                           output, input, block_offsets, filter,
                           input.valid != nullptr, stream);
 
+    // TODO: TEMPORARY FIX
```

Comment:
I'm working on a PR for this, including adding tests for stream compaction on cudf::nvstring_category columns.

Reply:
Great! This is only a placeholder change

Reply:
Removed